roles/acm/utils/tasks/monitor-install.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Get cluster install status
+  kubernetes.core.k8s_info:
+    api: extensions.hive.openshift.io/v1beta1
+    kind: AgentClusterInstall
+    name: "{{ utils_cluster_name }}"
+    namespace: "{{ utils_cluster_namespace }}"
+  register: install_status
```

Comment:
don't we need to add the `utils` prefix to the variables we define in the tasks?